aptos-move/aptos-vm/src/aptos_vm.rs

```diff
@@ -644,6 +647,18 @@ impl AptosVM {
         senders: Vec<AccountAddress>,
         script_fn: &EntryFunction,
     ) -> Result<SerializedReturnValues, VMStatus> {
+        let is_friend_or_private = session.load_function_def_is_friend_or_private(
```

Comment:
I'm a little bit confused by the semantics here. To whom are we saying that the entry point is a friend of?

Reply:
To a Rust native, in particular the `is_safe_call` native.

Reply:
We just need to know if it's private `entry` but `public(friend) entry` will work as well, since it cannot be wrapped into a script? Or if it is wrapped into a script, the bit here won't be set. I'll add a test case for this.

Reply:
Indeed, it cannot be wrapped in a script, so can't even test for it.